.github/workflows/shellcheck.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Is there any shell left to check now that you've gotten rid of the test shell scripts?